confectory-core/src/main/java/net/obvj/confectory/Confectory.java

```diff
@@ -0,0 +1,49 @@
+package net.obvj.confectory;
+
+/**
+ * A default {@link ConfigurationContainer}
+ * 
+ * @author @Dautomne (Fernando Tiannamen)
+ * @since 0.1.0
+ */
+public final class Confectory
+{
+
+    /**
+     * The initial {@link ConfigurationContainer} to be loaded as default
+     */
+    protected static ConfigurationContainer DEFAULT_CONFIGURATION_CONTAINER;
+
+    {
+        DEFAULT_CONFIGURATION_CONTAINER = new ConfigurationContainer();
+    }
+
+    /**
+     * Sets an already existent {@link ConfigurationContainer} as Default
+     *
+     * @param pConfigurationContainer the default {@link ConfigurationContainer} to be set
+     */
+    public Confectory(ConfigurationContainer pConfigurationContainer)
+    {
+        DEFAULT_CONFIGURATION_CONTAINER = pConfigurationContainer;
+    }
```

Comment:
We want the Facade to be fully static. We don't want any instances of this class being created.

(1) Make the constructor private and throw an exception (see an example of similar classes in the util package)
(2) Create a static setter method `setDefaultContainer`
(3) Please don't use the "p" prefix on method parameters